mizar/obj/tests/test_pod_arktos_vpc1.yaml

```diff
@@ -3,12 +3,10 @@ kind: Pod
 metadata:
   name: arktospodvpc1net1
   labels:
-    arktos.futurewei.com/network: vpc1
+    arktos.futurewei.com/network: my-net-mizar-1
   annotations:
-    arktos.futurewei.com/nic: "[{\"name\": \"eth0\", \"ip\": \"12.2.0.4\", \"subnet\": \"net1\"}]"
+    arktos.futurewei.com/nic: "[{\"name\": \"eth0\", \"ip\": \"20.0.0.14\", \"subnet\": \"net0\"}]"
 spec:
   containers:
   - name: arktospodvpc1net1
-    image: localhost:5000/testpod
-    ports:
-      - containerPort: 443

```

Comment:
My impression is that we need this when we test pod connectivity by curl. Please confirm.

Reply:
curl can work with other ports as well. the default port is 80